source/extensions/filters/http/cache/cache_headers_utils.cc

```diff
@@ -200,8 +200,18 @@ bool VaryHeader::hasVary(const Http::ResponseHeaderMap& headers) {
   return vary_header != nullptr && !vary_header->value().empty();
 }
 
+namespace {
+// The separator characters are used to create the vary-key, and must be characters that are
+// invalid to be inside values and header names. The chosen characters are invalid per:
+// https://tools.ietf.org/html/rfc2616#section-4.2.
+// Used to separate the values of different headers.
+const std::string header_separator = "\n";
+// Used to separate multiple values of a same header.
+static const std::string in_value_separator = "\r";
```

Comment:
`static` has no effect here

Reply:
Ah, I forgot to remove it. I'm changing it either way per the other comment above.